kernels/tiled_mult/Makefile

```diff
@@ -0,0 +1,45 @@
+# Courtesy of Federico Ficarelli
+
+.DEFAULT_GOAL := all
+
+include ../../runtime/snax-mac.rules
+include ../../runtime/Makefile.rules
+
+TESTS =
+TESTS += baseline.x
+TESTS += untiled.acc_dialect.x
```

Comment:
where tiled?

Reply:
and what is the difference in baseline vs untiled? Does having a baseline in C make sense?

Reply:
Nope, removed!